domain/src/main/java/gov/cms/ab2d/domain/User.java

```diff
@@ -7,7 +7,7 @@
 import javax.persistence.*;
 
 @Entity
-@Table(name = "`user`")
+@Table(name = "user_account")
 @Getter
 @Setter
 @EqualsAndHashCode(onlyExplicitlyIncluded = true)
```

Comment:
I only renamed the table. 
@dkrylovsb, Should I have renamed the entity to `UserAccount` as well to match the new table name?

Reply:
Entity can stay I guess

Reply:
Okay.  Will leave the entity name unchanged.